torchtitan/parallelisms/__init__.py

```diff
@@ -19,8 +19,9 @@
 models_parallelize_fns = {
     "llama2": parallelize_llama,
     "llama3": parallelize_llama,
+    'opt': parallelize_llama,
```

Comment:
could we change the name of parallelize_llama to parallelize_decoder_only? This looks like it's a bug even if it's not.